docs/quickstarts/react.mdx

```diff
@@ -3,34 +3,85 @@ title: Use Clerk with React
 description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your React application.
 ---
 
-import {Steps} from "nextra-theme-docs";
-import { Callout } from "nextra-theme-docs";
-import { YouTube } from "@/components/Youtube";
-import { Images } from "@/components/Images";
-import {Cards} from "@/components/Cards";
-import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+<TutorialHero 
+  quickstart="react"
+  exampleRepo={[ 
```

Comment:
I can make example Repo accept more than one repositories, and update the design to follow.
This way, we can include the "React Quickstart Repo" as well as "React + TypeScript Quickstart Repo"

Reply:
"React + TypeScript Quickstart Repo" - if this just an example name you came up with? I don't think this is a thing.

Reply:
oh, sorry, I saw a branch on that repo that looked like it was meant to serve as the repository but typed - this one: https://github.com/clerk/clerk-react-quickstart/tree/react-typescript-with-vite

Reply:
Gotcha. That branch is just the result of running the vite command to create the app. I don't think we need to link to it.

Basically the repo is:
- `main`, which has the `react-typescript-with-vite` and `integrate-clerk-with-react` merged into it.
- `integrate-react-router-dom-using-data-router-method` which is for the router guide.

To that end I think a link to the repo for the QS works and then a link to the branch for the Router guide works. Maybe something about merging the branch for the latter guide.